packages/uswds-utilities/src/styles/rules/opacity.scss

```diff
@@ -11,6 +11,9 @@ output:
 example:
   .opacity-0 {
     opacity: 0; }
+
+  .opacity-30 {
+    opacity:0.3; }
```

Comment:
Missing a space 
```suggestion
  .opacity-30 {
    opacity: 0.3; }
```

Reply:
This was updated in [267fc6d](https://github.com/uswds/uswds/pull/5859/commits/267fc6d3e58a1be2aaff128aeca4f8e6fad024b8)